### PR TITLE
Adapt titan QC flags and use alternative usage plotting with leaflet

### PR DESCRIPTION
**Pull request by trygveasp**

### CODE REVIEW

File: src/plots/plots_maps.R [hunk 1]
```diff
@@ -583,35 +598,68 @@ drawGriddedScattergeoTrace <- function(
...
+  "TITAN: blacklisted"="black", "TITAN: nometa"="lightblue",
+  "TITAN: plaus"="lightgreen", "TITAN: clim"="darkblue",
+  "TITAN: buddy"="lightgray", "TITAN: SCT"="darkred",
+  "TITAN: DEM"="lightgray", "TITAN: isol"="blue",
+  "TITAN: fg"="orange", "Fraction"="beige"
+  )
+  domains = c("Active", "Active(2)", "Blacklisted", "NA",   "Rejected", "Passive", "Rejected in TITAN", "Domain",    "TITAN: blacklisted", "TITAN: nometa", "TITAN: plaus", "TITAN: clim", "TITAN: buddy", "TITAN: SCT", "TITAN: DEM", "TITAN: isol", "TITAN: fg", "Fraction")
+  pallete = colorFactor(colors, domain=domains)
+
+   if ( F ) {
```
Comment (paulovcmedeiros) [inline src/plots/plots_maps.R:616]: Hi @trygveasp ,

I'm going through your code now and wonder why the `if ( F )` here. Unless this is some feature of the R-language I'm now aware of, the code in this block is never going to be executed.

Could you give me some context for why you've chosen this?

Thanks a lot in advance.

**Comment by trygveasp** [comment]
it's just to keep the old method as a reference and make it easy to switch back and forth. Maybe it should be a general switch or maybe we just remove it.

**Comment by paulovcmedeiros** [comment]
Oh, I see. Then I know you meant for it not to be executed. I'll look closer and probably will remove it (just to make it less confusing in the future if someone else wants to edit the code).

Thanks a lot for the clarification. And thanks for the changes too.

**Comment by paulovcmedeiros** [comment]
I have now merged this request into `master` with some very minor changes (just removed commented code, an inactive block a did some small refactoring).

Thanks a lot for this, @trygveasp .

**Pull request closed without merging**